tests/jvm/src/test/resources/metacp.expect

```diff
@@ -310,50 +310,58 @@ Schema => SemanticDB v3
 Uri => com/javacp/MetacJava.class
 Text => empty
 Language => Java
-Symbols => 25 entries
+Symbols => 31 entries
 
 Symbols:
 _root_. => package _root_
 _root_.com. => package com
 _root_.com.javacp. => package javacp
-_root_.com.javacp.MetacJava# => class MetacJava.{+6 decls}
+_root_.com.javacp.MetacJava# => class MetacJava.{+8 decls}
```

Comment:
This is out of sync with javacp. Do you think that should be a blocker, or we can open a ticket for the future?

Reply:
Good catch! Looking at the diff it seems metac emits too duplicate symbols for inner classes, which is a bug but not a blocker

```diff
============================
_root_.com.javacp.MetacJava#
============================
--- metac
+++ metacp
     declarations: "_root_.com.javacp.MetacJava#`<init>`()."
-    declarations: "_root_.com.javacp.MetacJava#Overload1#"
-    declarations: "_root_.com.javacp.MetacJava#Overload1#"
-    declarations: "_root_.com.javacp.MetacJava#Overload2#"
-    declarations: "_root_.com.javacp.MetacJava#Overload2#"
     declarations: "_root_.com.javacp.MetacJava#overload(A+1)."
     declarations: "_root_.com.javacp.MetacJava#overload(A+2)."
-    declarations: "_root_.com.javacp.MetacJava#StaticInner#"
-    declarations: "_root_.com.javacp.MetacJava#StaticInner#"
-    declarations: "_root_.com.javacp.MetacJava#Overload3#"
-    declarations: "_root_.com.javacp.MetacJava#Overload3#"
     declarations: "_root_.com.javacp.MetacJava#overload(A+3)."
+    declarations: "_root_.com.javacp.MetacJava#Overload2#"
+    declarations: "_root_.com.javacp.MetacJava#Overload3#"
+    declarations: "_root_.com.javacp.MetacJava#Overload1#"
+    declarations: "_root_.com.javacp.MetacJava#StaticInner#"
 accessibility {
```

Tracked in #1411